config/locales/idv/en.yml

```diff
@@ -5,24 +5,28 @@ en:
       activate_by_mail: Activate by mail
       activate_by_phone: Activate by phone
       cancel: Cancel and return to your profile
```

Comment:
Should this be "Cancel and return to your account"?

Reply:
I don't remember having a discussion about profile vs account - but I'm ok with calling this "account" 👍